Sources/SwiftDocC/Model/DocumentationNode.swift

```diff
@@ -421,6 +422,49 @@ public struct DocumentationNode {
             semantic.httpResponsesSectionVariants[.fallback] = HTTPResponsesSection(responses: responses)
         }
         
+        // The symbol ppossible values.
+        let symbolAllowedValues = symbol?.mixins[SymbolGraph.Symbol.AllowedValues.mixinKey] as? SymbolGraph.Symbol.AllowedValues
```

Comment:
minor: The symbol is known to be non-`nil` because of [this precondition](https://github.com/swiftlang/swift-docc/blob/main/Sources/SwiftDocC/Model/DocumentationNode.swift#L334). Also, the symbol has a subscript to make it more convenient to access mixins:
```suggestion
        // The property list symbol's allowed values.
        let symbolAllowedValues = symbol![mixin: SymbolGraph.Symbol.AllowedValues.self]
```